lib/Common/Common/NumberUtilities.h

```diff
@@ -4,6 +4,14 @@
 //-------------------------------------------------------------------------------------------------------
 #pragma once
 
+enum LikelyType
```

Comment:
Since this enum is used in a number of places, can we give it a more descriptive name (`LikelyNumberType`?) and make it an `enum class` so that its values are namespaced?

Reply:
good info: https://stackoverflow.com/questions/18335861/why-is-enum-class-preferred-over-plain-enum